docs/_documentation/fundamentals/Customizing-using-App-and-Setup.md

```diff
@@ -0,0 +1,537 @@
+---
+layout: documentation
+title: Customizing using App and Setup
+category: Fundamentals
+order: 10
+---
+
+# Customizing using App and Setup
+
+In each deployed MvvmCross application there are two key classes which control how your app starts:
+
+* the App class in the core project - which provides the initialization for your core business logic and your viewmodels.
+* the Setup class in the native UI project - this Setup is a bootstrapper for the MvvmCross system and for your app.
+
+## App.cs
+
+Typically App.cs provides only initialization of:
+
+* simple rule-based IoC registration - e.g.:
+
+```c#
+      CreatableTypes()
```

Comment:
There is a default tab space for almost all code examples. Can you remove the first tab? After that it look good to merge